src/Core/BasePlugin.php

```diff
@@ -77,6 +77,13 @@ class BasePlugin implements PluginInterface
      */
     protected $configPath;
 
+    /**
+     * The templates path for this plugin.
+     *
+     * @var string
+     */
+    protected $templatePath;
```

Comment:
Is this also supported as a constructor option?

Reply:
I didn't notice other paths could be passed as constructor option. Will update to allow setting template path too.

Reply:
Done.